kong/plugins/jwt/schema.lua

```diff
@@ -28,6 +28,7 @@ return {
     anonymous = {type = "string", default = "", func = check_user},
     run_on_preflight = {type = "boolean", default = true},
     maximum_expiration = {type = "number", default = 0, func = check_positive},
+    header_name = {type = "string", default = "authorization"},
```

Comment:
both the `uri_param_names` and the `cookie_names` accept an array. So it makes sense to also add `header_names` as an array